src/whylogs/__init__.py

```diff
@@ -5,6 +5,11 @@
 from .app.session import reset_default_session
 from .mlflow import enable_mlflow
 
+# noinspection PyProtectedMember
```

Comment:
Since WhyLogs is a library (not just an application) we should not configure the loggers during import.  Logging config and control should be handled by any running application, and should be do-able within the standard python logging framework.

My preference is to provide a convenience utility to easily enable logging for interactive workflows, something like:

`whylogs.display_logging()`

We could allow users to configure this to get called in their environment at whylogs import time, if we really want.